lib/app_files/category_i_files/dashboard.dart

```diff
@@ -49,16 +49,47 @@ class _DashboardState extends State<Dashboard> {
                   child: Row(
                     mainAxisAlignment: MainAxisAlignment.end,
                     children: [
+                      Builder(
+                          builder: (BuildContext context) {
+                            return Container(
+                              width: 70, // Set the desired width
```

Comment:
sized should be dynamic. use the dimentions.dart page to refer to the sizes. if you don't have a specific width or height. create it.